### PR TITLE
Implement Bootstrap home layout

### PR DESCRIPTION
**Pull request by NaMLiM**
Implement Bootstrap home layout to master

### CODE REVIEW

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 13
           

Complexity increasing per file
==============================
- js/home.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690&bid=17696113)

File: css/home.css [hunk 1]
```diff
@@ -2,217 +1,170 @@
...
+.dropdown:hover>.dropdown-menu {
+  display: block;
 }
 
-.sambutan {
-  display: grid;
-  grid-auto-flow: column;
-  grid-gap: 10%;
-  background-color: lightgray;
+.dropdown>.dropdown-toggle:active {
```
Comment (NaMLiM) [inline css/home.css:62]: Codacy Issue found: [Expected whitespace before "*/" (comment-whitespace-inside)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 2]
```diff
@@ -2,217 +1,170 @@
...
-  padding-left: 20%;
+.dropdown:hover>.dropdown-menu {
+  display: block;
 }
 
-.sambutan {
-  display: grid;
-  grid-auto-flow: column;
-  grid-gap: 10%;
-  background-color: lightgray;
```
Comment (NaMLiM) [inline css/home.css:96]: Codacy Issue found: [Expected single space before ">" (selector-combinator-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 3]
```diff
@@ -2,217 +1,170 @@
...
 
-.navlink a:hover {
-  background-color: #eaba3a;
-  color: #fff;
+.dropdown-menu a:hover {
+  background: #eaba3a;
 }
 
-.ceo {
-  padding-left: 20%;
```
Comment (NaMLiM) [inline css/home.css:89]: Codacy Issue found: [Expected single space before ">" (selector-combinator-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 4]
```diff
@@ -2,217 +1,170 @@
...
+  transition: all 700ms;
 }
 
-.nav-2 {
-  display: grid;
-  min-height: 50px;
-  grid-auto-flow: column;
-  grid-gap: 20px;
+.bg-custom.scrolled {
+  transform: translateY(0px);
```
Comment (NaMLiM) [inline css/home.css:32]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 5]
```diff
@@ -2,217 +1,170 @@
...
-  grid-gap: 10%;
-  background-color: lightgray;
+.dropdown>.dropdown-toggle:active {
+  /*Without this, clicking will make it sticky*/
+    pointer-events: none;
 }
 
-.kata-sambutan {
-  display: inline;
-  padding: 45px;
```
Comment (NaMLiM) [inline css/home.css:101]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css (hunk 3 above)
Comment (NaMLiM) [inline css/home.css:89]: Codacy Issue found: [Expected single space after ">" (selector-combinator-space-after)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 6]
```diff
@@ -2,217 +1,170 @@
...
 
-.footer1 div .link a img {
-  width: 220px;
+.footer2 {
+  background-color: #020e28;
 }
 
-.sosmed {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (NaMLiM) [inline css/home.css:201]: Codacy Issue found: [Expected selector ".grayLink a" to come before selector ".dropdown-menu a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: js/home.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+var $a = document.getElementById("nav");
+window.onscroll = function() {scrollFunction()};
```
Comment (NaMLiM) [inline js/home.js:2]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css (hunk 2 above)
Comment (NaMLiM) [inline css/home.css:96]: Codacy Issue found: [Expected single space after ">" (selector-combinator-space-after)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 8]
```diff
@@ -2,217 +1,170 @@
...
+  background: transparent !important;
+  transition: all 700ms;
 }
 
-.nav-2 {
-  display: grid;
-  min-height: 50px;
-  grid-auto-flow: column;
-  grid-gap: 20px;
+.bg-custom.scrolled {
```
Comment (NaMLiM) [inline css/home.css:31]: Codacy Issue found: [Unexpected unit (length-zero-no-unit)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: js/home.js (hunk 7 above)
Comment (NaMLiM) [inline js/home.js:2]: Codacy Issue found: ['scrollFunction' was used before it was defined.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 9]
```diff
@@ -2,217 +1,170 @@
...
-.nav-1 a {
-  text-decoration: none;
+.flag-icon {
+  height: 20px;
 }
 
-.nav-1 a img {
-  width: 20px;
+.bg-custom {
+  transform: translateY(20px);
```
Comment (NaMLiM) [inline css/home.css:26]: Codacy Issue found: [Use of !important](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 10]
```diff
@@ -2,217 +1,170 @@
...
+  display: block;
 }
 
-.sambutan {
-  display: grid;
-  grid-auto-flow: column;
-  grid-gap: 10%;
-  background-color: lightgray;
+.dropdown>.dropdown-toggle:active {
+  /*Without this, clicking will make it sticky*/
```
Comment (NaMLiM) [inline css/home.css:63]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690&bid=17696113)

File: css/home.css [hunk 11]
```diff
@@ -1,218 +1,170 @@
...
-.footer1 div .link a img {
-  width: 220px;
+.footer2 {
+  background-color: #020e28;
 }
 
-.sosmed {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
+.grayLink a,
```
Comment (NaMLiM) [inline css/home.css:157]: Codacy Issue found: [Expected selector ".grayLink a" to come before selector ".dropdown-menu a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css [hunk 12]
```diff
@@ -1,218 +1,170 @@
...
 }
 
-.sambutan {
-  display: grid;
-  grid-auto-flow: column;
-  grid-gap: 10%;
-  background-color: lightgray;
+.dropdown > .dropdown-toggle:active {
+  /* Without this, clicking will make it sticky */
+    pointer-events: none;
```
Comment (NaMLiM) [inline css/home.css:64]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

File: css/home.css (hunk 11 above)
Comment (NaMLiM) [reply css/home.css:157]: Ignore

**Comment by NaMLiM** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690&bid=17696113)

File: css/home.css [hunk 13]
```diff
@@ -1,218 +1,170 @@
...
-.footer1 div .link a img {
-  width: 220px;
+.footer2 {
+  background-color: #020e28;
 }
 
-.sosmed {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr 1fr;
+.grayLink a,
```
Comment (NaMLiM) [inline css/home.css:157]: Codacy Issue found: [Expected selector ".grayLink a" to come before selector ".dropdown-menu a:hover" (no-descending-specificity)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5398690)

Comment (NaMLiM) [reply css/home.css:157]: Ignore

**Review by kanam129 — APPROVED** [review]
siip

**Comment by kanam129** [comment]
ok gan

**Pull request merged**